### PR TITLE
New ARM Template for Multiple Windows VM with Custom Script

### PR DESCRIPTION
**Pull request by mahendra-shinde**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 101-multiple-windows-vms-with-common-script/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+                "az-500.ps1",
+                "container-lab.ps1",
+                "container-vs2019-lab.ps1"
+            ],
+            "metadata": {
+                "description": "Choose script to launch from list of available scripts"
+            }
+        }
+    },
+    "functions": [],
```
Comment (bmoore-msft) [inline 101-multiple-windows-vms-with-common-script/azuredeploy.json:76]: remove empty property

File: 101-multiple-windows-vms-with-common-script/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    "variables": {
+        "strname": "[toLower(concat('labvmstg',uniqueString(resourceGroup().id)))]",
+        "scriptUrl": "[uri(parameters('_artifactsLocation'),concat(parameters('scriptFilename'),parameters('_artifactsLocationSasToken')))]"
+    },
+    "resources": [
+        {
+            "name": "[variables('strname')]",
```
Comment (bmoore-msft) [inline 101-multiple-windows-vms-with-common-script/azuredeploy.json:83]: See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

File: 101-multiple-windows-vms-with-common-script/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+        "vmCount": {
+            "type": "int",
+            "defaultValue": 2,
+            "metadata": {
+                "description": "Number of LAB-VM tobe deployed"
+            }
+        },
+        "dns-prefix": {
+            "type": "string",
+            "defaultValue": "labvm-400",
```
Comment (bmoore-msft) [inline 101-multiple-windows-vms-with-common-script/azuredeploy.json:15]: this default must be globally unique

File: 101-multiple-windows-vms-with-common-script/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+                    "commandToExecute": "[concat('powershell -ExecutionPolicy Bypass -file  ', parameters('scriptFilename'))]"
+                }
+            },
+            "copy": {
+                "name": "scriptCopy",
+                "count": "[parameters('vmCount')]"
+            }
+        }
+    ],
+    "outputs": {
```
Comment (bmoore-msft) [inline 101-multiple-windows-vms-with-common-script/azuredeploy.json:294]: remove empty property

File: 101-multiple-windows-vms-with-common-script/azuredeploy.parameters.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "_artifactsLocation": {
```
Comment (bmoore-msft) [inline 101-multiple-windows-vms-with-common-script/azuredeploy.parameters.json:5]: remove the parameters that have defaultValues and use GEN values for the rest (see contribution guide)

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 101-multiple-windows-vms-with-common-script/azuredeploy.parameters.json (hunk 5 above)
Comment (mahendra-shinde) [reply 101-multiple-windows-vms-with-common-script/azuredeploy.parameters.json:5]: done

File: 101-multiple-windows-vms-with-common-script/azuredeploy.json (hunk 4 above)
Comment (mahendra-shinde) [reply 101-multiple-windows-vms-with-common-script/azuredeploy.json:294]: done

File: 101-multiple-windows-vms-with-common-script/azuredeploy.json (hunk 3 above)
Comment (mahendra-shinde) [reply 101-multiple-windows-vms-with-common-script/azuredeploy.json:15]: done

File: 101-multiple-windows-vms-with-common-script/azuredeploy.json (hunk 2 above)
Comment (mahendra-shinde) [reply 101-multiple-windows-vms-with-common-script/azuredeploy.json]: done

File: 101-multiple-windows-vms-with-common-script/azuredeploy.json (hunk 1 above)
Comment (mahendra-shinde) [reply 101-multiple-windows-vms-with-common-script/azuredeploy.json]: done

File: 101-multiple-windows-vms-with-common-script/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+        "dns-prefix": {
+            "type": "string",
+            "defaultValue": "labvm-400",
+            "metadata": {
+                "description": "DNS Prefix used by LabVM Public IP"
+            }
+        },
+        "adminUser": {
+            "type": "string",
+            "defaultValue": "mahendra",
```
[the remark this answers is not in the document]
Comment (mahendra-shinde) [reply 101-multiple-windows-vms-with-common-script/azuredeploy.json]: done

File: 101-multiple-windows-vms-with-common-script/azuredeploy.json [hunk 7]
```diff
@@ -0,0 +1,299 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+
```
[the remark this answers is not in the document]
Comment (mahendra-shinde) [reply 101-multiple-windows-vms-with-common-script/azuredeploy.json]: done

**Pull request merged**